.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: c
+sudo: required
+dist: trusty
```

Comment:
Do we have to mention the distribution here? Can we make it work without that? It would be better not to care about distro updates on Travis CI.